qt/applications/workbench/workbench/app/workbench_launch_wrapper.py

```diff
@@ -0,0 +1,23 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+import subprocess
+import sys
+
+from workbench.app.main import main
+
+
+def launch():
```

Comment:
I think this is missing the bits for return codes. The final lines should be like [this](https://github.com/neutrons/SNAPRed/blob/next/src/snapred/__main__.py#L139-L140).